chinglee-iot/FreeRTOS
# Test cases for pvTaskGetThreadLocalStoragePointer

Description
-----------

added test cases for pvTaskGetThreadLocalStoragePointer

Test Steps
-----------

cd FreeRTOS/Test/CMock   
make coverage

Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
